homeassistant/components/imeon_inverter/strings.json

```
@@ -77,6 +97,25 @@
       "inverter_injection_power_limit": {
         "name": "Injection power limit"
       },
+      "manager_inverter_state": {
+        "name": "Inverter state",
+        "state": {
+          "unsynchronised": "Unsynchronised",
+          "grid_consumption": "Grid consumption",
+          "grid_injection": "Grid injection",
+          "grid_synchronised_but_not_used": "Grid unsynchronised but used"
```

For the default UI of Home Assistant we use international English:
```suggestion
          "unsynchronised": "Unsynchronized",
          "grid_consumption": "Grid consumption",
          "grid_injection": "Grid injection",
          "grid_synchronised_but_not_used": "Grid unsynchronized but used"
```

For the UK-English variant the "z" will be replaced with an "s".